crowbar_framework/app/assets/stylesheets/branding.scss

```diff
@@ -1,2 +1,15 @@
 @import "bootstrap/variables";
 @import "bootstrap/mixins";
+#nodegroups {
+  .panel>.list-group .list-group-item, .panel>.panel-collapse>.list-group .list-group-item {
+    border: 1px solid #ddd;
+    border-radius: 5px; margin: 5px;
+    box-shadow: 1px 1px 0px #eee
+  }
+  .panel-heading+.list-group .list-group-item:first-child {
+    border-top-width: 1px;
+  }
+  .panel>.list-group:last-child .list-group-item:last-child, .panel>.panel-collapse>.list-group:last-child .list-group-item:last-child{
```

Comment:
Selector should have depth of applicability no greater than 2, but was 5<br>Each selector in a comma sequence should be on its own single line<br>Opening curly brace `{` should be preceded by one space